config.m4

```diff
@@ -41,6 +41,8 @@ if test "$PHP_PDO_SNOWFLAKE" != "no"; then
   LDFLAGS="$LDFLAGS $SNOWFLAKE_CLIENT_DIR/deps-build/linux/curl/lib/libcurl.a"
   LDFLAGS="$LDFLAGS -Wl,--no-whole-archive"
 
+  CFLAGS="-std=c99 $CFLAGS"
```

Comment:
For PDO only at the moment. Some debug macro requires C99 features. Will drop this if this doesn't work on other platform.